src/tidalforce_odom/src/odom.py

```diff
@@ -127,8 +128,13 @@ def update(self, left, right):
     rospy.init_node("odom")
 
     #ip = rospy.get_param("~ip", "roboRIO-1721-FRC")
+    #ip = rospy.get_param("~ip", "localhost") # For testing
     #ip = rospy.get_param("~ip", "10.17.21.55") # First mDNS almost never works on the feild
-    ip = rospy.get_param("~ip", "localhost") # For testing
+    if (len(sys.argv) > 1): # If there is more than one argument
```

Comment:
you don't actually need the outer parenthesis.

Reply:
So I should just use `if len(sys.argv) > 1:` right?